ui/components/src/test/inputs/QuantityInput.test.tsx

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as React from "react";
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import { IModelApp, MockRender, QuantityType } from "@bentley/imodeljs-frontend";
+import TestUtils from "../TestUtils";
+import { QuantityInput } from "../../ui-components/inputs/QuantityInput";
+import { SpecialKey } from "@bentley/ui-abstract";
+
+describe("QuantityInput", () => {
+  const rnaDescriptorToRestore = Object.getOwnPropertyDescriptor(IModelApp, "requestNextAnimation")!;
+  function requestNextAnimation() { }
+
+  before(async () => {
+    // Avoid requestAnimationFrame exception during test by temporarily replacing function that calls it.
+    Object.defineProperty(IModelApp, "requestNextAnimation", {
+      get: () => requestNextAnimation,
+    });
+    await TestUtils.initializeUiComponents();
+    await MockRender.App.startup();
+  });
+
+  after(async () => {
+    await MockRender.App.shutdown();
+    TestUtils.terminateUiComponents();
+    Object.defineProperty(IModelApp, "requestNextAnimation", rnaDescriptorToRestore);
+  });
+
+  afterEach(cleanup);

```

Comment:
Not necessary as cleanup is already done in 
https://github.com/imodeljs/imodeljs/blob/c0b588d4b2ae8d9600dc10a3fe33ba9d326e1681/ui/scripts/setup-tests.js#L103-L104